mtp_cashbook/apps/cashbook/tasks.py

```diff
@@ -87,3 +95,26 @@ def credit_individual_credit_to_nomis(user, user_session, credit_id, credit):
             html_template='cashbook/email/credited-confirmation.html',
             anymail_tags=['credited'],
         )
+
+
+NOMIS_ACCOUNTS = {'cash', 'spends', 'savings'}
+
+
+@spoolable()
+def check_balance_is_below_cap(prison, prisoner_number):
+    # NB: balances are not known in private estate currently, but cashbook is not used in there
+    try:
+        nomis_account_balances = nomis.get_account_balances(prison, prisoner_number)
```

Comment:
This is pretty similar to the checking done in the API, I wonder if we should make this a common method between the two?

Reply:
it's a combination of both the api and the send-money code so it can't entirely be replaced with a call to the api. it is a slight repetition, i agree. when we decide what to do with the outstanding issues regarding capping, we could maybe consolidate all of that within mtp-common?

Reply:
Sure, I don't think we need to consolidate it as part of this ticket